app/Store.php

```diff
@@ -25,4 +25,12 @@ function prefecture()
     {
         return $this->belongsTo('App\Prefectue');
     }
+
+    function bestreview()
+    {
+        return $this->hasMany('App\Review')
+        ->orderBy('reviews', 'desc')
+        ->orderBy('created_at', 'desc')
+        ->get();
```

Comment:
評価が高い&最新の口コミを上位1件だけ取得したいので、ここはfirstを使いましょう！
```suggestion
        ->first();
```